src/sphinx/reference/event-log.rst

```diff
@@ -223,9 +223,9 @@ Disaster recovery is executed per ``ReplicationEndpoint`` by calling its asynchr
 During execution of disaster recovery, directly connected endpoints must be available. These are endpoints for which replication connections have been configured at the endpoint to be recovered. Availability is needed because connected endpoints need to update internal metadata before they can resume event replication with the recovered endpoint.
 
 .. hint::
-   The ``Future[Unit]`` returned by ``recover()`` completes when the internal metadata update completed. For actually starting the replication of missing events from other locations to the endpoint being recovered, it must be activated with ``activate()``.
+   In theory an ``ReplicationEndpoint`` could always be activated at the beginning by calling its ``recover()`` method, however as this requires that all directly connected endpoints must be available it is not recommended.
```

Comment:
In theory, **a** ... . However, as this ... available, it is not recommended. 

Sorry for nitpicking :smiley: